crates/bootstrap_srv/src/space.rs

```diff
@@ -0,0 +1,228 @@
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex};
+
+use crate::store::*;
+
+/// A map of spaces.
+#[derive(Clone)]
+pub struct SpaceMap(Arc<Mutex<HashMap<bytes::Bytes, Space>>>);
```

Comment:
Is there a reason to use `bytes::Bytes` instead of the wrapper type you made `SpaceId`?

Reply:
Yes. I see this server / protocol being useful for WAN discovery outside of projects using kitsune itself. Perhaps it would actually be better to exist outside this monorepo, but that is more overhead for us. At the very least, I didn't want this to have any dependencies on other kitsune crates.

Reply:
In that case do you want to define a SpaceId or similar wrapper type in this crate just for readability?

Reply:
Sure, I can start with just a typedef for code clarity. I don't think there's any need for a newtype yet.

Reply:
Done: https://github.com/holochain/kitsune2/pull/10/commits/263d5ab3f9e742b19c46ce7a11d370205ad904ae